### DISCUSSION TITLE
5ghz option won't work any more

### DISCUSSION THREAD

**Discussion by josehdx** [Mygod/VPNHotspot#406] [Q&A]
Hello lovely dev

Feel free to close my issue if you don't find anything wrong with my logs

But time ago a was able to use the 5ghz spectrum and now I got an error when I try to force this option

I think my wifi card is probably damaged

Take a look and let me know

Regards

[logcat_08-06-2022_03-07-50.txt](https://github.com/Mygod/VPNHotspot/files/9274182/logcat_08-06-2022_03-07-50.txt)

**Comment by josehdx** [comment]
Dear dev. I hope you read my topic. I did a test with an old version. 2.10.9. i have set the hotspot to work only with 5gz and everytime i restart the hotspot, it works as intended. I dont remember in what version started to fail..i can do some manual testing but if you can check the previous log and let me know what is failing i can try to do some other tests if required. Thanks

**Comment by Mygod** [comment]
What error?

**Comment by josehdx** [comment]
"failed to create p2p group"

Now is solved but my solution was weird. I started to update the app from 2.10.9 to the next and lates increment i.e. 2.11x , 2.12 etc and even the latest (2.15.2) works fine. I just want to add a side note. On repeater setting, on the AP band option, i currently see the "2.4/5ghz" option at the top , and then all the frequencies. When i was having the error, it happened either when i selected any 5ghz band manually, or when i selected the only "5gz" band. I dont see the latest 5gz band as an option anymore, so i guess there was some kind of AP List messing around. Additionally, my AP is set as only 5ghz on the setting of my phone (red magic 5s) but this was set like this before. I will keep it in mind in case it happens again.  Thanks

**Reply by Mygod** [reply]
You might have turned on/off repeater safe mode.

**Reply by josehdx** [reply]
I think i tried multiple options (even flashing original/FIrst stock ROM) so probably the settings in the phone were messed as well. But im happy that now it works. Thanks

**Comment by josehdx** [comment]
Ok. I just noticed something. After some time after rebooting the phone the 5ghz networks stop working on my phone. Only 2.4ghz is available but after reboot, and immediately launching the hotspot app, it works again with 5ghz. Weird thing

**Comment by josehdx** [comment]
Ok. I just noticed something. After some time after rebooting the phone the 5ghz networks stop working on my phone. Only 2.4ghz is available but after reboot, and immediately launching the hotspot app, it works again with 5ghz. Weird thing
